themes/bootstrap3/templates/search/advanced/layout.phtml

```diff
@@ -103,7 +103,7 @@
             <?php endif; ?>
             <div id="group<?=$group ?>" class="adv-group">
               <div class="adv-group-terms">
-                <label class="adv-group-label" for="search_lookfor<?=$group ?>_0"><?=$this->transEsc("adv_search_label")?>:</label>
+                <label class="adv-group-label" <?=$this->transEsc("adv_search_label")?>:</label>
```

Comment:
search_lookfor_n_0 already has an aria-label directly on it, so associating this label as well causes confusion.

Reply:
The `for` attribute allows users to click on the label to focus the input. It shouldn't interfere with aria or other screen reader features.

Reply:
Also, an extra `>` was trimmed here.

Reply:
Thanks for catching the missing `>` -- I have restored it. Regarding the `for` attribute, WAVE reports an error because some elements have multiple labels associated with them. I understand the focus behavior may be useful, but it does seem problematic to me to have two different labels for the same thing -- and this label doesn't really describe the first search box but rather the entire region... so I wonder if we've been abusing the semantics of the tag to get the functionality we desire. Maybe there's a better way?

Reply:
I think the issue is actually that the template also contains a `for` to the first element of the first group (as a placeholder). This is certainly a solution for bootstrap3. Perhaps a refactor of this template in the new theme can help restore this convenience.